tests/e2e/smoketest-with-jaeger/02-generate-traces.yaml

```diff
@@ -7,7 +7,7 @@ spec:
     spec:
       containers:
         - name: tracegen
-          image: ghcr.io/open-telemetry/opentelemetry-collector-contrib/tracegen:latest
+          image: ghcr.io/open-telemetry/opentelemetry-collector-contrib/tracegen:v0.70.0
```

Comment:
I know is not part of this task but, could you replace this by `telemetrygen`?

Reply:
sure, done.